### PR TITLE
Fix player mob direction while riding the janicart and other rideable things

### PR DESCRIPTION
**Pull request by NIXC**
:cl: NIN
fix: Player now faces correctly when riding Janicart and other ride able things
/:cl:

Pretty quick fix, just forces the players mob to face in the same direction as the thing they are riding. This shouldn't break unless something exists that the player doesn't face forward while riding on.

### CODE REVIEW

**Comment by nicbn** [comment]
Is this in documentation a typo or a meme?
Yes.

File: code/datums/riding.dm [hunk 1]
```diff
@@ -86,6 +86,7 @@
 		if(!Process_Spacemove(direction) || !isturf(ridden.loc))
 			return
 		step(ridden, direction)
+		user.dir = ridden.dir //Why this isnt checked somewhere else is byond me
```
Comment (MrPerson) [inline code/datums/riding.dm]: This should probably use setDir()

Comment (RemieRichards) [inline code/datums/riding.dm]: no probably about it, definitely ```setDir()```

**Comment by RemieRichards** [comment]
This is was a regression, when Bman and I updated ```handle_vehicle_offsets()``` to support multi-riders we accidentally removed a ```setDir()```

Though logically, it does fit better where you've put it now (since if it was were it was before then multi-riders would be everyone in a vehicle faces forwards, where as where you've put it passengers can have their own dirs which makes sense)

**Pull request closed without merging**